USFMToolsSharp/Models/Markers/VMarker.cs

```diff
@@ -17,8 +17,14 @@ public override string PreProcess(string input)
             VerseNumber = match.Groups[1].Value;
             return match.Groups[2].Value;
         }
+        public void setVerseCharacter(string NumChar)

```

Comment:
As before this should be a property that pulls from the contents of it directly (See note in the html renderer)